frontend/pages/auth/google.tsx

```diff
@@ -0,0 +1,68 @@
+import AppContext from '@/components/Context/TopLevelContext';
```

Comment:
i presume this is the page that receives the URL? based on reading the backend code.

looks good in my own opinion

Reply:
Yep